### DISCUSSION TITLE
How to download Tailwind CSS Table Plugin?

### DISCUSSION THREAD

**Discussion by montasim** [STRINGLABITSOLUTIONS/Bhojon#196] [Q&A] [answered]
In short.

**Comment by d4tdev** [comment, marked as answer]
### With NPM 
npm install tailwindcss-tables --save
 
### With Yarn 
yarn add tailwindcss-tables
 
### Manually 
Create a /plugins/tailwindcss-tables folder in your project and drop 'index.js' inside of it.

See details at: https://www.npmjs.com/package/tailwindcss-tables

**Reply by montasim** [reply]
@d4rtj congratulations. Now you have a galaxy brain badge.

**Reply by d4tdev** [reply]
thanks so much 💚

**Reply by montasim** [reply]
You are welcome bro.
